<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stockPage</class>
 <widget class="QWidget" name="stockPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Складской учет</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="chkShowOnlyCurrentAndDescendantsStock">
     <property name="text">
      <string>Отображать остатки только текущего и нижестоящих складов</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCheckBox" name="chkPermitRequisitionsOnlyParentStock">
     <property name="text">
      <string>Оформлять требования только на вышестоящий склад</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QCheckBox" name="chkShowMainStockRemainings">
     <property name="text">
      <string>Отображать остатки основного склада</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="cmbAccordingRequirementsType">
     <item>
      <property name="text">
       <string>Никакие</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Только на основной склад</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Все</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblAccordingRequirementsStock">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Согласовывать требования</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chkShowOnlyLsInFilterNomenklature">
     <property name="text">
      <string>Отображать только ЛС в фильтре номенклатуры</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkShowMainStockRemainings</tabstop>
  <tabstop>chkPermitRequisitionsOnlyParentStock</tabstop>
  <tabstop>chkShowOnlyCurrentAndDescendantsStock</tabstop>
  <tabstop>cmbAccordingRequirementsType</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
